Graphics/Formats/Assimp/Anim.hsc

```diff
@@ -68,13 +70,17 @@ instance Storable Animation where
   alignment _ = #alignment aiAnimation
   peek p = do
     mName            <- liftM aiStringToString $ (#peek aiAnimation, mName) p
+    logPrint mName
     mDuration        <- (#peek aiAnimation, mDuration) p
+    logPrint mDuration
     mTicksPerSecond  <- (#peek aiAnimation, mTicksPerSecond) p
-    mNumChannels     <- (#peek aiAnimation, mNumChannels) p
+    logPrint mTicksPerSecond
+    mNumChannels     <- (#peek aiAnimation, mNumChannels) p :: IO CUInt
+    logPrint mNumChannels
     mChannels'       <- (#peek aiAnimation, mChannels) p
-    mChannels        <- peekArray mNumChannels mChannels'
-    mNumMeshChannels <- (#peek aiAnimation, mNumMeshChannels) p
+    mChannels        <- peekArray (fromIntegral mNumChannels) mChannels'
+    mNumMeshChannels <- (#peek aiAnimation, mNumMeshChannels) p :: IO CUInt
```

Comment:
i just did this for choosing the right fromIntegral instances (happened a couple of times). More improvements on the way ;)